optionals/compat_spe/CfgVehicles/turrets.hpp

```diff
@@ -0,0 +1,128 @@
+class All {
+    class EventHandlers;
+};
+class LandVehicle;
+class StaticWeapon: LandVehicle {
+    class ACE_Actions {
+        class ACE_MainActions;
+    };
+    class UserActions;
+};
+
+// --- Mortars -----------------------------------------------------------------
+
+class StaticMortar: StaticWeapon {};
+class SPE_StaticMortar_base: StaticMortar {
+    class Turrets: Turrets {
+        class MainTurret: MainTurret {};
+    };
+    class EventHandlers: EventHandlers {
+        class SPE_StaticWeaponsHandler {
+            init = "";
+        };
+    };
+};
```

Comment:
```
class All {
    class EventHandlers;
};
class AllVehicles: All {};
class Land: AllVehicles {};
class LandVehicle: Land {};
class StaticWeapon: LandVehicle {
    class ACE_Actions {
        class ACE_MainActions;
    };
    class Turrets {
        class MainTurret;
    };
    class UserActions;
};

// --- Mortars -----------------------------------------------------------------
class StaticMortar: StaticWeapon {
    class Turrets: Turrets {
        class MainTurret: MainTurret {};
    };
};
class SPE_StaticMortar_base: StaticMortar {
    class Turrets: Turrets {
        class MainTurret: MainTurret {};
    };
    class EventHandlers: EventHandlers {
        class SPE_StaticWeaponsHandler {
            init = "";
        };
    };
};
```
I think this is what we need, not tested

Reply:
Ah yeah I forgot to circle back to that, wanted to check where they were defined

Reply:
I tested Pabst's changes, worked perfectly